docs/zh/openmldb_sql/deployment_manage/ONLINE_REQUEST_REQUIREMENTS.md

```diff
@@ -57,15 +57,19 @@ SELECT substr(COL7, 3, 6) FROM t1;
 
 ### 在线请求模式下 `LAST JOIN` 的使用规范
 
-- 仅支持`LAST JOIN`类型。
-- 至少有一个JOIN条件是形如`left_source.column=right_source.column`的EQUAL条件，**并且`right_source.column`列需要命中右表的索引（key 列）**。
-- 带排序LAST JOIN的情况下，`ORDER BY`只支持单列的列引用表达式，列类型为 int16, int32, int64 or timestamp, **并且列需要命中右表索引的时间列**。
-- 右表 TableRef
+1. 仅支持`LAST JOIN`类型。
+2. 至少有一个JOIN条件是形如`left_source.column=right_source.column`的EQUAL条件，**并且`right_source.column`列需要命中右表的索引（key 列）**。
+3. 带排序LAST JOIN的情况下，`ORDER BY`只支持单列的列引用表达式，列类型为 int16, int32, int64 or timestamp, **并且列需要命中右表索引的时间列**。满足条件 2 和 3 的情况我们简单称做表能被 LAST JOIN 的 JOIN  条件优化
```

Comment:
ts column type should be bigint or timestamp?

Reply:
the five types is supported in SQL engine, is there any difference in deployment ?

Reply:
Yes, if you deploy, creating index will be failed

Reply:
that's indeed index's restriction, anyway I've remove the two types in case confusing.